src/components/TagWallFilter/Filter/Filter.js

```diff
@@ -135,7 +135,9 @@ class Filter extends React.Component {
             className={`${namespace}__list-item__entry`}
             aria-labelledby={itemProps.id}
           >
-            {itemText}
+            <span className={`${carbonPrefix}text-truncate--end`}>
```

Comment:
Similar comment to https://github.com/carbon-design-system/ibm-security/pull/513/files#r416481413